454 domain docs (#456)
18F/federalist.18f.gov: pages/documentation/customer-responsibilities.md

#### You own your content

> Federalist provides templates for you to start with in configuring your sites, but is not responsible for editing or updating the content or local configuration of your site. The Federalist team ensures that the publishing mechanism remains available to you so that your content edits can be published nearly immediately. Your content must be low impact according to [FIPS 199](http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.199.pdf) and each branch on GitHub is published publicly by Federalist. Federalist is not suitable for hosting the following types of information:

Reviewer: ```suggestion
Federalist provides templates for you to start with in configuring your sites, but is not responsible for editing or updating the content or local configuration of your site. The Federalist team ensures that the publishing mechanism remains available to you so that your content edits can be published within minutes. Your content must be low impact according to [FIPS 199](http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.199.pdf) and each branch on GitHub is published publicly by Federalist. Federalist is not suitable for hosting the following types of information:
```

- PII (Personally Identifiable Information)
- FOUO (For Official Use Only)
- CUI (Confidential Unclassified Information)
- Classified Information
- other content that otherwise requires authorization to view

Federalist suggests that you add your point of contact to your repo with editing rights for troubleshooting purposes, but this is not required.

#### You own your code

You may wish to use Jekyll plugins, Google Analytics from GSA's [Digital Analytics Program](https://www.digitalgov.gov/services/dap/) using embedded Javascript, or other code on your site. Federalist makes this possible - we use analytics extensively on our sites - but you are responsible for the security and stability of any custom code.

You are responsible for ensuring compliance with any and all additional federal regulations, including the [21st Century IDEA](https://digital.gov/resources/21st-century-integrated-digital-experience-act/).

#### You own your domain and DNS

> Federalist does not manage your domain name or provide DNS service. To launch a site on Federalist, you must configure the DNS for your domain to point to a domain provided by the Federalist team (meaning, that visitors to that address will be pointed at Federalist so they can load your site from us). Setting DNS for a new or existing domain may involve working with other offices at your agency; these processes typically vary.

Reviewer: ```suggestion
Federalist does not manage your domain name nor provide DNS services. To launch a site on Federalist, you must configure the DNS for your domain to point to a domain provided by the Federalist team (meaning, that visitors to that address will be pointed at Federalist so they can load your site from us). Setting DNS for a new or existing domain may involve working with other offices at your agency; these processes typically vary.
```

> If your domain not an apex (e.g. 2nd level) domain, the process may be more challenging as some DNS providers do not support all required DNS record types. You will need to plan a solution before signing an agreement, please see [custom domains](/documentation/custom-domains) for more details.

Reviewer: ```suggestion
If your domain is not an apex (e.g. 2nd level) domain, the process may be more challenging as some DNS providers do not support all required DNS record types. We recommend that you plan a solution before signing an agreement, please see [custom domains](/documentation/custom-domains) for more details.
```
